python-client/build.gradle.kts

```diff
@@ -80,6 +80,12 @@ tasks {
         command = "run test-fast"
     }
 
+    create<PythonTask>("full-python-test") {
```

Comment:
I'd rename it to `test-all` to make it "python" independent in case we do the same separation for backend tests and in this case `gradle test-all` would trigger all tests in both: java and python